### DISCUSSION TITLE
weight measures

### DISCUSSION THREAD

**Discussion by stm-tmd** [tcgoetz/GarminDB#165] [General]
I'm using this script to get my garmin data in golden cheetah and that works very nice, so thank you for that :-)

But what I'm missing is some weight-data. There is the "normal" weight, but I would also like to track things like BMI, fat%, muscle mass, bone mass and water%. 
I've got a garmin scale which measures all these things and sends them to garmin-connect, so I think this could also be collected with this script.

Is this possible, am I missing something, or is this not an option?

Thanks in advance
Guido

**Comment by tcgoetz** [comment]
This is the code that import the weight data into the database: https://github.com/tcgoetz/GarminDB/blob/master/garmindb/import_monitoring.py#L25. If you look in your HealthData directory you will find a weight directory and files in it names weight_{date}.json. You can see what values are available there. The import code and the matching database code would need to be enhanced to add additional values.

**Reply by stm-tmd** [reply]
Thanks, I'll look into it.

**Comment by EmptySoft** [comment]
since 2 days i have a Garmin Index S2, in the Weight Directory are the .json Files. weight,bmi, bodyFat, bodyWater, boneMass, muscleMass and a lot more are there. Schould i send you one File for testing?

**Reply by skinnie** [reply]
Sorry to dig this, was the garmin index s2 ever supported?
